secretsanta/cli/cli.py

```diff
@@ -0,0 +1,58 @@
+import click
+import json
+
+from secretsanta.main import funs as secretsanta
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
```

Comment:
```suggestion

@click.group(context_settings={'help_option_names':['-h', '--help']})
```
Doesn't look like we need to define `CONTEXT_SETTINGS` and `{}` are considered more pythonic than `dict()`.

Reply:
agreed, we have just one util (`group`) so makes sense to simplify